softpack_core/service.py

```diff
@@ -448,6 +452,89 @@ async def buildStatus(  # type: ignore[no-untyped-def]
             ),
         }
 
+    @staticmethod
+    @router.post("/createEnvironment")
```

Comment:
Better to use REST convention of lowercase with hyphens for all these?
https://restfulapi.net/resource-naming/